content/ideas/co-design-session-3/index.md

```diff
@@ -0,0 +1,144 @@
+Duration: 20 minutes**
+
+A Sidewalk representative provided an overview of the accessibility principles developed by Sidewalk Labs. The principles were grouped into three main areas; digital, physical and general accessibility. Printed copies of the principles and examples were provided to each group for review.
+
+### Design Activity: Part 2 - Brainstorming
+**Objective: Brainstorming ideas<br>
+Duration: 30 minutes**
+
+After sharing personal experiences relevant to the theme assigned to their group and also learning about the accessibility principles, each group was given a challenge related to their theme to resolve. Groups had 30 minutes to brainstorm a pool of ideas to address their challenge. The challenges for each theme are presented below:
+
+In & Out:
+
+* Challenge: Imagine a better experience in the future for moving in and out of an office
+building
+
+Up & Down:
+
+* Challenge: Imagine a better experience in the future for moving between different levels
+
+Getting Around:
+
+* Challenge: Imagine a better experience in the future for switching modes of transportation
+
+As the groups engaged in discussion, the assigned facilitators provided insight and guided the discussion to reflect on each theme. Each group then had the opportunity to briefly share their ideas with the larger group and received feedback before prototyping their idea the next day. The end of the group discussion on preliminary ideas marked the end of the first half of this two-day session. Participants were encouraged to reflect on their ideas and be more observant of the thresholds over the following day. 
+
+{{< imgblock src="images/IMG_7848.jpg" resize="wide" alt="Pictured above are the ideas generated by different groups around personal encounters with thresholds in the city, and reflections of positive and negative experiences associated with these encounters. The responses listed are from groups \"Hedgehog\", \"The A team\",\"Challenging\", \"Cats and Dogs\", \"Group #2\" and \"Assorted Barrier Busters\"" >}}
+
+## DAY 2
+
+### Design Activity: Part 3 - Prototype Challenge
+
+**Objective: Building a prototype to meet unrepresented needs<br>
+Duration: 1 hour 30 minutes**
+
+The second day began promptly with the prototyping challenge. Participants were given an hour and a half to choose an idea they brainstormed the previous day, expand on the idea, and build or prepare a demonstration of their idea. Each group was provided some prompting questions to help this process: Is it accessible? Is it safe? Does it support independence? What would you need? How can technology help? Who else is involved?
+
+{{< imgblock src="images/IMG_7863.jpg" resize="wide" alt="Pictured is group \"Group #2\" prototype. They have used lego blocks to build a platform, craft paper to build an elevator cube and plasticine to create a mobility device, such as an electric scooter made and a guide dog." >}}
```

Comment:
Currently waiting to hear back from @sepidehshahi regarding this group's name. I left a comment about this in the Google doc used to create this post.